test/junit/scala/CollectTest.scala

```diff
@@ -52,15 +52,18 @@ class CollectTest {
   def testStreamCollectFirst(): Unit =
     testing(Seq(1), Some(1))(Stream.continually(1) collectFirst { case x if f(x) && x < 2 => x})
 
-  @Ignore @Test
+  @Disabled
+  @Test
```

Comment:
a consistent order for these annotations would be preferred

```suggestion
  @Test
  @Disabled
```